### PR TITLE
feat: suppress default branches configuration for release-branch master

### PR DESCRIPTION
**Pull request by acazacu**

### CODE REVIEW

**Comment by merlinnot** [comment]
What issue does it solve? I'd like to understand and test the before/after behavior.

**Comment by acazacu** [comment]
> What issue does it solve? I'd like to understand and test the before/after behavior.

It allows semantic-release to run on it's default branches configuration, something which right now we suppress via defining a default for the action's `release-branch` parameter.

**Comment by merlinnot** [comment]
So their default configuration is different than what we've defined? Is it a breaking change to the API? Should we enhance our documentation with this information?

**Comment by acazacu** [comment]
> So their default configuration is different than what we've defined? Is it a breaking change to the API? Should we enhance our documentation with this information?

It is not a breaking change, I have updated the README.md file to highlight the new default.

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ steps:
     uses: ridedott/release-me-action@master
     with:
       # Configure semantic release to run from a specific branch.
-      # Default: master
+      # Default: @semantic-release/semantic-release Javascript API branches parameter default value
```
Comment (merlinnot) [inline README.md]: Can we add some link to their documentation, so it's easier to find?

Comment (acazacu) [reply README.md]: Done!

**Pull request merged**